hamilton/version.py

```diff
@@ -1 +1 @@
-VERSION = (1, 46, 0)
+VERSION = (1, 45, 0)
```

Comment:
remove this change.

Reply:
aren't we on 1.46.0 ?

Reply:
yep we're on 1.46.0 this was reverting that.

Reply:
so remove this too.